packages/golden-layout/src/LayoutManager.ts

```diff
@@ -369,6 +370,35 @@ export default class LayoutManager extends EventEmitter {
     this.emit('initialised');
   }
 
+  /**
+   * Adds a react child to the layout manager
+   * @param id Unique panel id
+   * @param element The React element
+   */
+  addReactChild(id: string, element: React.ReactNode) {
+    this._reactChildren.set(id, element);
+  }
+
+  /**
+   * Removes a react child from the layout manager
+   * @param id Unique panel id
+   */
+  removeReactChild(id: string) {
+    this._reactChildren.delete(id);
+  }
+
+  /**
+   * Gets the react children in the layout
+   *
+   * Used in @deephaven/dashboard to mount the react elements
+   * inside the app's React tree
+   *
+   * @returns The react children to mount for this layout manager
+   */
+  getReactChildren() {
+    return [...this._reactChildren.values()];
+  }
+
```

Comment:
I'll check if I can. I think initially I tried it, but ran into issues when opening a panel w/ the same name since we don't assign a new key to that panel (e.g. re-run a code block to recreate a table). But that might have been before I found where I needed to use `replaceItem` instead of `addItem` and `removeItem`